moveit_core/transforms/CMakeLists.txt

```diff
@@ -3,13 +3,16 @@ set(MOVEIT_LIB_NAME moveit_transforms)
 add_library(${MOVEIT_LIB_NAME} src/transforms.cpp)
 set_target_properties(${MOVEIT_LIB_NAME} PROPERTIES VERSION "${${PROJECT_NAME}_VERSION}")
 
-target_link_libraries(${MOVEIT_LIB_NAME} ${catkin_LIBRARIES} ${urdfdom_LIBRARIES} ${urdfdom_headers_LIBRARIES} ${Boost_LIBRARIES})
-add_dependencies(${MOVEIT_LIB_NAME} ${catkin_EXPORTED_TARGETS})
+target_link_libraries(${MOVEIT_LIB_NAME} ${rclcpp_LIBRARIES} ${rmw_implementation_LIBRARIES} ${urdfdom_LIBRARIES} ${urdfdom_headers_LIBRARIES} ${Boost_LIBRARIES})
+# add_dependencies(${MOVEIT_LIB_NAME} ${catkin_EXPORTED_TARGETS})
+ament_target_dependencies(${MOVEIT_LIB_NAME}
+            tf2_eigen)
+
 
 install(TARGETS ${MOVEIT_LIB_NAME}
-        LIBRARY DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
-        ARCHIVE DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION})
-install(DIRECTORY include/ DESTINATION ${CATKIN_GLOBAL_INCLUDE_DESTINATION})
+        LIBRARY DESTINATION lib
+      ARCHIVE DESTINATION lib)
+install(DIRECTORY include/ DESTINATION include)
```

Comment:
It is weird to me that the majority of ROS2 packages use the paradigm `install(DIRECTORY include/${PROJECT_NAME}/ DESTINATION include/${PROJECT_NAME})` if `install(DIRECTORY include/ DESTINATION include)` works just as well. Is there a reason why we wouldn't want to use `install(DIRECTORY include/moveit/transforms/ DESTINATION include/moveit/transforms/)` here?

Reply:
You guys are the experts. I'd say we'd rather keep it simple and use `install(DIRECTORY include/ DESTINATION include)`.

Reply:
aside from hard-coded paths and as long as there is no other local headers folder that should not be exposed as public API, your simple approach should be alright @vmayoral .

The only downside I can see is that `lint`ing the build description (is there an `ament_lint`?) becomes harder if you do not include the package name in the destination.

From an interface-perspective I would prefer 
```
install(DIRECTORY include/${PROJECT_NAME} DESTINATION include)
```
because this installs the requested include path (e.g. `#include <${PROJECT_NAME}/my_header.hpp>`
without by any chance renaming it.

Reply:
@vmayoral appears to do the same thing to me, but cleaner. i like it! but im not expert on ament standards

Reply:
My question was more philosophical than practical. @vmayoral's version is standard across ROS2 so I am happy to leave this as is.